src/content/en/2021/security.md

```diff
@@ -1099,54 +1104,58 @@ We now turn to cryptominer market share in the following figure.
 
 Our results suggest that cryptojacking is still a serious attack vector, and necessary measures should be used for it.
 
-Note that not all of these websites are infected. Website operators may also deploy this technique (instead of showing ads) to finance their website. But the legal use of this technique is also heavily discussed technically, legally, and ethically.
+Note that not all of these websites are infected. Website operators may also deploy this technique (instead of showing ads) to finance their website. But the use of this technique is also heavily discussed technically, legally, and ethically.
 
 Please also note that our results may not show the actual state of the websites infected with cryptojacking. Since we run our crawler once a month, not all websites that run cryptominer can be discovered. This is the case, for example, if a website remains infected for only X days and not on the day our crawler ran.
 
 ## `security.txt`
 
-<a hreflang="en" href="https://datatracker.ietf.org/doc/html/draft-foudil-securitytxt-12">`security.txt`</a> is a well-known file format for websites to provide a standard for vulnerability reporting. Website providers can provide contact details, PGP key, policy, and other information in this file. White hat hackers can then use this information to conduct security analyses on these websites or report a vulnerability. The following figure shows that 5% of the websites already use this standard—although it is a very young standard.
+<a hreflang="en" href="https://datatracker.ietf.org/doc/html/draft-foudil-securitytxt-12">`security.txt`</a> is a file format for websites to provide a standard for vulnerability reporting. Website providers can provide contact details, PGP key, policy, and other information in this file. White hat hackers can then use this information to conduct security analyses on these websites or report a vulnerability.
 
 {{ figure_markup(
   image="security-usage-of-well-known-security.png",
   caption="Use of `security.txt`.",
-  description="Bar chart showing 4.93% of websites in desktop and 4.89% of websites in mobile have `security.txt` endpoint.",
+  description="Bar chart showing 4.9% of websites in desktop and 4.9% of websites in mobile have `security.txt` endpoint.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR38ZfiZXxdGuzG4ywCEPKIU1Wl6E2bZwcQPyALavGq5q948gmWb8sT-Xo5T6K5z8smKPg6EKxV0JUI/pubchart?oid=1958147881&format=interactive",
   sheets_gid="399976976",
   sql_file="well-known_security.sql"
   )
 }}
 
-And the following figure shows that the policy is the most used property of utilized `security.txt` files.
+We see that just under 5% of the websites already use this standard—although it is still a very young standard. We hope to see this number increase in future, since having a `security.txt` shows that the website administrators care about security and are willing to work with white hat hackers to improve security of their website.
```

Comment:
```suggestion
We see that just under 5% of the websites already use this standard—even though it is still a very young standard. We hope to see this number increase in future, since having a `security.txt` is an indication that the website administrators care about security and are willing to work with vulnerability reporters to improve security of their website.
```

Suggest tweaking this slightly @SaptakS